File: arrayBasedProblems/problem_1.py
# Problem 1 : How do you find the largest and smallest number in an array of 1-100?

# 1st Way
# array =[]
# for x in range(1,100+1):
#     array.append(x)

# for x in range(0,len(array)):
#     for y in range(1,len(array)):
#         if(array[y] < array[y-1]):
#             temp = array[y]
#             array[y] = array[y-1]
#             array[y-1] = temp

# print(f"The largest number in an array is {array[len(array)-1]}")
# print(f"The smallest number in an array is {array[0]}")


# 2nd Way
array =[4,5,8,9,1,4,7,40,2,0,2,3]
newArray = sorted(array)

print(f"The largest number in an array is {newArray[len(newArray)-1]}")
print(f"The smallest number in an array is {newArray[0]}")

File: arrayBasedProblems/problem_1.js
// Problem 1 : How do you find the largest and smallest number in an array of 1-100?
var array = [4,8,6,9,1,2,5,6,8,7,1,3,858,-8,0,699,1452]
for(var i=0;i<array.length;i++){
    for(j=i;j<array.length;j++){
        if(array[i]>array[j]){
            temp = array[i]
            array[i] = array[j]
            array[j] = temp
        }
    }
}
console.log(`The smallest value is: ${array[0]}`)
console.log(`The latgest value is: ${array[array.length-1]}`)
